CONTRIBUTING.md

```diff
@@ -10,7 +10,22 @@ We actively welcome your pull requests.
 5. Make sure your code lints.
 6. If you haven't already, complete the Contributor License Agreement ("CLA").
 
-## Contributor License Agreement ("CLA")
+### Forking, cloning and running Rapid
+
+- Make sure to use the latest `node` with something like  `nvm install node`
+- Run `npm run all`
+- Run `npm run quickstart` to start the server
+
+### Debugging and testing
+
+- In VSCode go to Run>Add Configuration... and select Launch Chrome
+- Examples of breakpoints
+- Conditional breakpoints
+- Some interesting files to break (init, etc)
```

Comment:
Considering that Visual Studio Code (VSCode) is not the sole IDE in use, it may be beneficial to omit this specific reference from the documentation.

Reply:
Absolutely, will add a note for it. Thanks!

Reply:
I've added a suggested method and alternatives to VSCode in case anybody wants to try their own toolset